Library/Homebrew/manpages/brew.1.md

```diff
@@ -438,7 +438,8 @@ Note that these flags should only appear after a command.
     Display the file or directory used to cache <formula>.
 
   * `--cellar`:
-    Display Homebrew's Cellar path. *Default:* `/usr/local/Cellar`
+    Display Homebrew's Cellar path. *Default:* `$(brew --prefix)/Cellar`, or if
+    that directory doesn't exist, `$(brew --repository)/Cellar`.
```

Comment:
@jacknagel: I'm open to another way to express this that doesn't involve shellisms.

@chdiza: If `brew` is being run from within a repository, `/usr/local/Cellar` is never examined. This I figured out from discussion with @tdsmith in #39429 and by reading the source.